.github/workflows/ci.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         uses: actions/checkout@v2.3.4
 
       - name: Install libusb
-        run: sudo apt install -y libusb-1.0-0-dev libftdi1-dev
+        run: sudo apt install -y libusb-1.0-0-dev libftdi1-dev pkg-config
```

Comment:
The issue is probably missing `libudev`, `pkg-config` should already be installed.
The following should work:

```yaml
- name: Install libusb, libudev and libftdi (linux)
        run: |
          sudo apt update
          sudo apt install -y libusb-1.0-0-dev libftdi1-dev libudev-dev
        # Only install on Ubuntu
        if: (matrix.os == 'ubuntu-latest') || (matrix.os == 'ubuntu-20.04')
```

(Taken from https://github.com/probe-rs/probe-rs/blob/master/.github/workflows/ci.yml)